.buildkite/pipeline.gpu.large.yml

```diff
@@ -18,6 +18,15 @@
     - ./ci/env/env_info.sh
     - bazel test --config=ci $(./ci/run/bazel_export_options) --build_tests_only --test_tag_filters=datasets_train doc/...
 
+- label: ":tv: :database: :dog: Jax Integration GPU Tests and Examples (Python 3.7)"
```

Comment:
Can we be consistent with the emojis between GPU and CPU tests?

Reply:
sounds good to me